user_docs/en/userGuide.md

```diff
@@ -300,21 +300,21 @@ When add-ons are installed, the Add-on Store opens to the "Installed add-ons" ta
 #### Available add-ons {#AvailableAddons}
 When the window first opens, add-ons may take a few seconds to load.
 NVDA will read the name of the first add-on once the list of add-ons finishes loading.
-Available add-ons are listed alphabetically in a grid.
+Available add-ons are listed alphabetically in a multi-column list.

```

Comment:
Better use the word "list" everywhere rather than "list" once, "grid" once and "table" another time.